### ISSUE TITLE
3.19.0: Image map: Commenting out image produces error

### ISSUE THREAD

**Issue by Moonbase59** [valentine195/obsidian-leaflet-plugin#149]
I have a working (3.18.4) 2-layer image map:

````
```leaflet
id: khorvaire
image:
    - [[Khorvaire.jpg|Khorvaire]]
    - [[Khorvaire-Talenta Plains2.png|Talenta Plains]]
height: 500px
lat: 50
long: 50
zoomDelta: 0.5
defaultZoom: 6.5
unit: miles
scale: 400
marker: Check, -3.8681, 6.6143
#marker: Current, 50, 50
```
````

Wanted to test the new image overlays, so I commented out one of the images:

````
```leaflet
id: khorvaire
image:
    - [[Khorvaire.jpg|Khorvaire]]
    #- [[Khorvaire-Talenta Plains2.png|Talenta Plains]]
height: 500px
lat: 50
long: 50
zoomDelta: 0.5
defaultZoom: 6.5
unit: miles
scale: 400
marker: Check, -3.8681, 6.6143
#marker: Current, 50, 50
```
````

Result:

````
```leaflet
There was an error rendering the map:

There was an error reading the image file: 
```
````

Console:

```text
2main.ts:669 Error: There was an error reading the image file: 
    at eval (eval at <anonymous> (app.js:1), <anonymous>:24574:19)
    at Generator.next (<anonymous>)
    at eval (eval at <anonymous> (app.js:1), <anonymous>:33:71)
    at new Promise (<anonymous>)
    at __awaiter (eval at <anonymous> (app.js:1), <anonymous>:29:12)
    at toDataURL (eval at <anonymous> (app.js:1), <anonymous>:24522:12)
    at ObsidianLeaflet.eval (eval at <anonymous> (app.js:1), <anonymous>:32161:38)
    at Generator.next (<anonymous>)
    at eval (eval at <anonymous> (app.js:1), <anonymous>:33:71)
    at new Promise (<anonymous>)
eval @ main.ts:669
rejected @ tslib.es6.js:74
Promise.then (async)
step @ tslib.es6.js:75
fulfilled @ tslib.es6.js:73
Promise.then (async)
step @ tslib.es6.js:75
fulfilled @ tslib.es6.js:73
Promise.then (async)
step @ tslib.es6.js:75
eval @ tslib.es6.js:76
__awaiter @ tslib.es6.js:72
postprocessor @ main.ts:131
(anonymous) @ app.js:1
t.postProcess @ app.js:1
t.postProcess @ app.js:1
h @ app.js:1
e.onRender @ app.js:1
requestAnimationFrame (async)
Be @ app.js:1
e.queueRender @ app.js:1
e.set @ app.js:1
t.set @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
s @ app.js:1
Promise.then (async)
l @ app.js:1
(anonymous) @ app.js:1
o @ app.js:1
t.setMode @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
o @ app.js:1
t.setState @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
o @ app.js:1
t.setViewState @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
o @ app.js:1
t.onSwitchView @ app.js:1
(anonymous) @ app.js:1
```

**Comment by Moonbase59** [comment]
More testing shows that:

```yaml
image: [[Khorvaire.jpg]]
```
works.

Any _indented_ array with just _one_ entry doesn’t:

```yaml
image:
  - [[Khorvaire.jpg]]
```

_Indented_ arrays with _more than one entry_ work:

```yaml
image:
  - [[Khorvaire.jpg]]
  - [[Khorvaire.jpg]]
```

**Comment by Moonbase59** [comment]
Seems fixed in 3.19.1
